src/exchange/exchange.py

```diff
@@ -40,7 +40,7 @@ class Exchange:
     provider: Provider
     model: str
     system: str
-    moderator: Moderator = field(default=ContextTruncate())
+    moderator: Moderator = field(default=ContextSummarizer())
```

Comment:
let's remove this default so there's no chance of setting it wrong (e.g. better to error due to not inheriting model than using wrong one)

```
class ContextSummarizer(ContextTruncate):
    def __init__(
        self,
        model: str = "gpt-4o-mini",
```

Reply:
yes, or it needs to be fixed to take in the correct model